src/styles/blocks/main.scss

```
@@ -0,0 +1,20 @@
+.products {
+  display: flex;
+  flex-wrap: wrap;
+  margin: 0 auto;
+  max-width: 944px;
+  padding-top: 50px;
+  padding-bottom: 48px;
+
+  &__card {
+    margin-right: 48px;
+
+    &:nth-child(4n) {
+      margin-right: 0;
+    }
+
+    &:nth-last-child(-n + 4) {
+      margin-top: 46px;
+    }
```

```suggestion
  &__card {
    margin-right: 48px;
     margin-bottom: 46px;

    &:nth-child(4n) {
      margin-right: 0;
    }

    &:nth-last-child(-n + 4) {
      margin-bottom: 0;
    }
```
this suggestion still performs well if you get additional rows of cards